docs/weaviate/manage-collections/multi-tenancy.mdx

```diff
@@ -51,7 +51,7 @@ Multi-tenancy is disabled by default. To enable multi-tenancy, set `multiTenancy
       text={TSCode}
       startMarker="// START EnableMultiTenancy"
       endMarker="// END EnableMultiTenancy"
-      language="ts"
+      language="tsindent"
```

Comment:
Oh what does this do? Why are we adding it only here?

Reply:
When we use block scopes in the code to redeclare the Weaviate client multiple times with the same name, for example, the code gets indented if a formatter is used. The `tsindent` styling removed that indent in the documentation. So far, I only formatted one file, but ideally, going forward, we will settle on a formatter to use across all the code snippets so they are all consistent.